### PR TITLE
Add supplier to SPDX ecosystem enrich

### PR DESCRIPTION
**Pull request by goneall**
fixes https://github.com/snyk/parlay/issues/76

Note: this PR does not include a unit test

### CODE REVIEW

**Review by mcombuechen — APPROVED** [review]
Thanks Gary! I will add some unit tests and create a new release after merging.

**Comment by mcombuechen** [comment]
@goneall is it possible for you to sign your commit, too?

File: lib/ecosystems/enrich_spdx.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/spdx/tools-golang/spdx"
 	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+	"github.com/spdx/tools-golang/spdx/v2/common"
```
Comment (mcombuechen) [inline lib/ecosystems/enrich_spdx.go:28]: This has to move one line above (bfore the v2.3 import).

Comment (goneall) [reply lib/ecosystems/enrich_spdx.go:27]: I just added a (signed) commit to move the import with should resolve this.

**Comment by goneall** [comment]
On my local git repo, all the commits are showing as signed off - did I miss a commit?

```
commit 22087483e67ed0ba0785fdce6e3c22b77110d0fb (HEAD -> upstreamissue76, origin/upstreamissue76)
Author: Gary O'Neall <gary@sourceauditor.com>
Date:   Wed Oct 23 17:54:09 2024 -0700

    Move import to resolve CI failure

    Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

commit 29f3e2aa2692f6a93f821477ca48ce27953a7d62
Author: Gary O'Neall <gary@sourceauditor.com>
Date:   Fri Oct 18 14:11:16 2024 -0700

    Add supplier to SPDX ecosystem enrich

    fixes https://github.com/snyk/parlay/issues/76

    Signed-off-by: Gary O'Neall <gary@sourceauditor.com>
```

**Comment by goneall** [comment]
I checked the [commits tab](https://github.com/snyk/parlay/pull/78/commits) for this pull request, and they all are showing signed off - so I'm not quite sure why the merging is blocked - @mcombuechen let me know if there is something I missed.

**Comment by mcombuechen** [comment]
Oh this is about the commit being cryptographically signed (GPG or similar). https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Comment by mcombuechen** [comment]
@goneall I'll close this PR in favor of #79, so we can get this fix out. I've attributed you shared authorship on the commit. I hope this works for you, let me know otherwise.

**Pull request closed without merging**